test/test_triton.py

```diff
@@ -0,0 +1,68 @@
+import logging
+import torch
+from torch import nn as nn
+import unittest
+
+import torch_xla.experimental.torch_triton as torch_triton
```

Comment:
One thing I forgot to mention. Do you think it's better to name it as triton instead of torch_triton?